### ISSUE TITLE
Why this vs deepcake/echo?

### ISSUE THREAD

**Issue by zommerfelds** [player-03/echoes#5]
> This framework was [designed and implemented by deepcake](https://github.com/deepcake/echo), and is now maintained by player-03.

This sentence makes me think that https://github.com/deepcake/echo is no longer maintained. Is this really true? Can we add more context? What is the story behind this repo? Why did you create this separate repo? Would be curious to know :)

EDIT: maybe we can add a note to `deepcake/echo` pointing here if it's really true that it is now maintained here.

**Comment by player-03** [comment]
Oh whoops, sorry for the late response. I didn't get a notification for this.

[In 2021, deepcake stated](https://github.com/deepcake/echo/pull/5#issuecomment-902132238) "Unfortunately, I no longer have any motivation or time to spend on this framework. And I would rather leave it as it is, with backward compatibility." In short, deepcake's version will no longer be updated, but I and [onehundredfeet](https://github.com/onehundredfeet/hmecs) are still maintaining our respective forks.

As for getting deepcake to endorse this fork? They would first have to decide between my and onehundredfeet's repos, and reviewing two repos requires time and motivation. But who knows, they might approve a PR linking to both options. Guess that's worth a shot.

**Comment by zommerfelds** [comment]
Thanks for the background, it's good to know!
I was using the original version, but could consider trying one of the forks.
Do you have any insight on your vs onehundredfeet's version?

If you don't get to it I can try to add your links to the deepcake repo. Cheers

**Comment by player-03** [comment]
> Do you have any insight on your vs onehundredfeet's version?

Working on it now, for the pull request.

tl;dr is that mine focuses on ease of use and issues that I've run into while developing single-player games. hmecs seems to be more focused on performance at scale.

---

Edit: I should also mention that this is part of my job, and so I've been able to put a lot more time into it than onehundredfeet has. Still not as much time as deepcake, but I'm probably going to get there at some point.

I estimated this by comparing the number of commits we made ([me vs. deepcake](https://github.com/player-03/echoes/graphs/contributors), [onehundredfeet vs. deepcake](https://github.com/onehundredfeet/hmecs/graphs/contributors)), and taking into account that my commits tend to be a lot larger than either of theirs. (I like to combine related changes into a single commit, so I don't have to write as many commit messages.)

**Comment by player-03** [comment]
I submitted a pull request (deepcake/echo#8). Does it look good?

**Comment by zommerfelds** [comment]
Sure, left a comment there. Thanks for following up!

**Comment by onehundredfeet** [comment]
People are welcome to take whatever they want from my fork.  It was deepcake who did the heavy lifting, I'm just gilding the lily.

**Comment by player-03** [comment]
Closing now that the pull request was accepted.
